src/components/Timeout/Timeout.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect, useRef } from 'react';
+import './timeout.css';
+
+export default function Timeout(props) {
+  //track idle time in background
+  //set time in minutes for idle time before modal should appear, and countdown time before auto logout
+  const idleTime = 1;
+  const countdownTime = 5;
+
+  const [idle, setIdle] = useState(false);
+
+  const time = useRef(0);
+
+  function reset() {
```

Comment:
good idea but no need to make this a helper since it's only one line

Reply:
Yeah I would agree, but this function is used more than once. I think it looks cleaner this way.

Reply:
you do you